packages/forms/src/Components/Section.php

```diff
@@ -48,12 +62,13 @@ public function heading($heading)
         return $this;
     }
 
-    public static function make($heading, $subheading = null, $schema = [])
+    public static function make($heading, $subheading = null, $schema = [], $collapsible = false)
```

Comment:
I don't know how I feel about having 4 arguments to a method... generally speaking, if I've got a method that has more than 3 arguments, there always a better way to do it. Everything becomes a bit too dense, especially when the previous argument is an array that's likely going to split the method call across multiple lines.

In this case, I think we can get rid of the `$collapsible` argument and just rely on the `collapsible` method instead. What do you think?

Reply:
This sounds good to me 👍

Reply:
Me too - happy with the collapsible() method